docs/reference/redirects.asciidoc

```diff
@@ -388,19 +380,19 @@ See <<commands>>.
 === Calendar resources
 
 See <<ml-get-calendar>> and
-{ml-docs}/ml-ad-run-jobs.html#ml-ad-calendars[Calendars and scheduled events].
+{ml-docs}/ml-ad-finding-anomalies.html[Calendars and scheduled events].
 
 [role="exclude",id="ml-filter-resource"]
 === Filter resources
 
 See <<ml-get-filter>> and
-{ml-docs}/ml-ad-run-jobs.html#ml-ad-rules[Machine learning custom rules].
+{ml-docs}/ml-ad-finding-anomalies.html[Machine learning custom rules].
```

Comment:
> This change should also be reverted, since the original is correct: https://www.elastic.co/guide/en/machine-learning/8.0/ml-ad-run-jobs.html#ml-ad-rules